apps/ejabberd/src/jlib.erl

```diff
@@ -26,7 +26,19 @@
 
 -module(jlib).
 -author('alexey@process-one.net').
-
+-xep([{xep, 59}, {version, "1.0"}]).
+-xep([{xep, 68}, {version, "1.2"}]).
+-xep([{xep, 18}, {version, "0.2"}]).
+-xep([{xep, 175}, {version, "1.2"}]).
+-xep([{xep, 157}, {version, "1.0"}]).
+-xep([{xep, 212}, {version, "1.0"}]).
+-xep([{xep, 114}, {version, "1.6"}]).
+-xep([{xep, 85}, {version, "2.1"}]).
```

Comment:
That's a client-side extension, the only server-side module that is aware of it is `mod_offline`, but I'm not sure if it should land there or in a "static" list of supported XEPs.